synapse/events/utils.py

```diff
@@ -49,7 +49,7 @@
 #       the literal fields "foo\" and "bar" but will instead be treated as "foo\\.bar"
 SPLIT_FIELD_REGEX = re.compile(r"(?<!\\)\.")
 
-CANONICALJSON_MAX_INT = (2**53) - 1
+CANONICALJSON_MAX_INT = (2 ** 53) - 1
```

Comment:
Autoformatted, hmmm. Do I need to update my `black` dependency locally?

Reply:
yup, this looks like the result of an old `black`.

Reply:
`pip install -e ".[all,dev]"` -> `Successfully installed black-22.3.0` ✅